packages/library-base/src/commonMain/kotlin/io/realm/kotlin/RealmConfiguration.kt

```diff
@@ -49,6 +50,10 @@ public interface RealmConfiguration : Configuration {
         schema: Set<KClass<out BaseRealmObject>>
     ) : Configuration.SharedBuilder<RealmConfiguration, Builder>(schema) {
 
+        init {
```

Comment:
Anything wrong with putting this in `Configuration.SharedBuilder`'s `init` block?